src/OpenSage.Game/GameWindow.cs

```diff
@@ -77,19 +64,37 @@ public GameWindow(string title, int x, int y, int width, int height, GraphicsBac
                 ResourceBindingModel = ResourceBindingModel.Improved
             };
 
+            var windowCreateInfo = new WindowCreateInfo(x, y, width, height, WindowState.Normal, title);
+
             if (preferredBackend != null)
             {
-                GraphicsDevice = AddDisposable(VeldridStartup.CreateGraphicsDevice(
-                    _window,
+                VeldridStartup.CreateWindowAndGraphicsDevice(
+                    windowCreateInfo,
                     graphicsDeviceOptions,
-                    preferredBackend.Value));
+                    preferredBackend.Value,
+                    out _window,
```

Comment:
Looks like you're missing a call to `AddDisposable` for the newly-created graphics device?

Reply:
I wasn’t sure how to make this to a disposable when it’s not a return type

Reply:
You can call `AddDisposable(_device);` as a separate statement.